addons/account_followup/account_followup.py

```diff
@@ -384,120 +349,106 @@ def _get_followup_overdue_query(self, cr, uid, args, overdue_only=False, context
-                                                    multi="followup",
-                                                    fnct_search=_payment_earliest_date_search),
-        }
 
-
-class account_config_settings(osv.TransientModel):
+    payment_responsible_id = fields.Many2one('res.users', ondelete='set null', string='Follow-up Responsible',
+                                             help="Optionally you can assign a user to this field, which will make him responsible for the action.",
+                                             track_visibility="onchange", copy=False)
+    payment_note = fields.Text('Customer Payment Promise', help="Payment Note", track_visibility="onchange", copy=False)
+    payment_next_action = fields.Text('Next Action', copy=False,
+                                help="This is the next action to be taken.  It will automatically be set when the partner gets a follow-up level that requires a manual action. ",
+                                track_visibility="onchange")
+    payment_next_action_date = fields.Date('Next Action Date', copy=False,
+                                help="This is when the manual follow-up is needed. "
+                                     "The date will be set to the current date when the partner "
+                                     "gets a follow-up level that requires a manual action. "
+                                     "Can be practical to set manually e.g. to see if he keeps "
+                                     "his promises.")
+    unreconciled_aml_ids = fields.One2many('account.move.line', 'partner_id', domain=['&', ('reconciled', '=', False), '&',
+                        ('account_id.deprecated','=', False), '&', ('account_id.internal_type', '=', 'receivable')])
+    latest_followup_date = fields.Date(compute='_get_latest',
+                                       help="Latest date that the follow-up level of the partner was changed",
+                                       store=False, multi="latest")
+    latest_followup_level_id = fields.Many2one('account_followup.followup.line', string="Latest Follow-up Level",
+        help="The maximum follow-up level", store=True, multi="latest")
+    latest_followup_level_id_without_lit = fields.Many2one('account_followup.followup.line', string="Latest Follow-up Level without litigation",
+        help="The maximum follow-up level without taking into account the account move lines with litigation",
+        store=True, multi="latest")
+    payment_amount_due = fields.Float(compute='_get_amounts_and_date',
+                                      string="Amount Due",
+                                      store=False, multi="followup",
+                                      search='_payment_due_search')
+    payment_amount_overdue = fields.Float(compute='_get_amounts_and_date',
+                                          string="Amount Overdue",
+                                          store=False, multi="followup",
```

Comment:
No need of store=False, Computed fields are not stored by default.
The 'multi' parameter does nothing in new API .